i18n/de.json

```diff
@@ -0,0 +1,61 @@
+{
+  "menu": {
+    "exit": "VERLASSEN",
+    "help": "HILFE",
+    "completed": "ABGESCHLOSSEN",
+    "language": "SPRACHE WÄHLEN",
```

Comment:
Der englische Begriff sollte erhalten bleiben, falls ein nicht-deutsch-sprachiger versehentlich in die falsche Sprache wechselt. Beispiel der Japanischen Uebersetzung: https://github.com/rvagg/workshopper/blob/master/i18n/ja.json#L6